presto-server-rpm/README.md

```diff
@@ -6,12 +6,12 @@ You can build an RPM package for Presto server and install Presto using the RPM.
 
 The RPM builds by default in Maven, and can be found under the directory `presto-server-rpm/target/`
 
-The RPM has a pre-requisite of Python >= 2.4. It also needs Oracle Java 1.8 update 40 (8u40 64-bit) pre-installed. The RPM installation will fail if any of these requirements are not
+The RPM has a pre-requisite of Python >= 2.6. It also needs Java 8 Update 161 or higher (8u161+), 64-bit pre-installed. Both Oracle JDK and OpenJDK are supported. The RPM installation will fail if any of these requirements are not
```

Comment:
AFAIR we support other JDKs now too.
@electrum can you suggest the wording?

Reply:
Maybe we should just remove this paragraph entirely, as it's yet another place to be out of date.